src/api/app/models/token/workflow.rb

```diff
@@ -7,14 +7,16 @@ def self.token_name
 
   def call(options)
     set_triggered_at
-    raise ArgumentError, 'A payload is required' if options[:payload].nil?
+    raise Token::Errors::MissingPayload, 'A payload is required' if options[:payload].nil?
```

Comment:
Raising a custom error to ensure that specs are valid. It would be possible to have a spec pass when calling `call` with too many arguments, even though this fictional spec wouldn't actually test this line.